src/components/pill/pill.js

```diff
@@ -92,6 +102,22 @@ class Pill extends React.Component {
     );
   }
 
+  get closeIcon() {
+    if (this.props.onDelete) {
+      return (
+        <Icon
+          className='carbon-pill__delete-icon'
+          data-element='close'
+          onClick={ this.props.onDelete }
+          type='cross'
+          onBlur={ this.onCloseIconBlur }
+          bgSize='small'
+        />
```

Comment:
Currently this won't be able to receive keyboard focus, so keyboard users won't be able to use the close icon.

You could wrap it in a `<button>` tag, which will add the correct behaviour for free (bar some CSS updates perhaps), or add an ARIA `role` attribute, `tabIndex` attribute, and event handlers to handle keyboard interaction.

Reply:
UX said they were thinking of going in a different direction than having the Icon tabbable so I removed the support for it. I've added it back for now as its something that can easily be removed when it's decided how they want it implemented